From efe1c9e539868c3a5175c1bc3a0d816a4eacdb2a Mon Sep 17 00:00:00 2001
From: Divya Narayanan Poojary <dnaray@codeaurora.org>
Date: Tue, 15 Nov 2016 19:16:17 +0530
Subject: [PATCH 15/19] legacy: let audioserver connect to thermal engine
 sockets

audioserver needs to access thermal engine for
speaker calibration.

Change-Id: I141e8962b6e39ba8407b6e54710c31b9574a1b2b
CRs-fixed: 1086008
---
 legacy-common/audioserver.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/legacy-common/audioserver.te b/legacy-common/audioserver.te
index 91d73e0..eff227b 100644
--- a/legacy-common/audioserver.te
+++ b/legacy-common/audioserver.te
@@ -4,6 +4,7 @@ unix_socket_send(audioserver, mpctl, mpdecision)
 unix_socket_connect(audioserver, mpctl, mpdecision)
 unix_socket_send(audioserver, mpctl, perfd)
 unix_socket_connect(audioserver, mpctl, perfd)
+unix_socket_connect(audioserver, thermal, thermal-engine)
 
 # Allow communication with peripheral manager
 use_per_mgr(audioserver)
-- 
2.7.4

